### PR TITLE
[New Rule] Suspicious Module Loaded by LSASS

### PR DESCRIPTION
**Pull request by Samirbous**
Converted an existing ER (diag) to DR with a good initial set of known FPs filtered.

### CODE REVIEW

File: rules/windows/credential_access_lsass_loaded_susp_dll.toml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+Identifies LSASS loading an unsigned or untrusted DLL. Windows Security Support Provider (SSP) DLLs are loaded into LSSAS 
+process at system start. Once loaded into the LSA, SSP DLLs have access to encrypted and plaintext passwords 
+that are stored in Windows, such as any logged-on user's Domain password or smart card PINs. 
+"""
+from = "now-9m"
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Module Loaded by LSASS"
+note = """## Setup
```
Comment (w0rk3r) [inline rules/windows/credential_access_lsass_loaded_susp_dll.toml:20]:
```suggestion
note = """## Setup

```

File: rules/windows/credential_access_lsass_loaded_susp_dll.toml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2022/12/28"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies LSASS loading an unsigned or untrusted DLL. Windows Security Support Provider (SSP) DLLs are loaded into LSSAS 
+process at system start. Once loaded into the LSA, SSP DLLs have access to encrypted and plaintext passwords 
+that are stored in Windows, such as any logged-on user's Domain password or smart card PINs.
```
Comment (w0rk3r) [inline rules/windows/credential_access_lsass_loaded_susp_dll.toml:13]:
```suggestion
Identifies LSASS loading an unsigned or untrusted DLL. Windows Security Support Provider (SSP) DLLs are loaded into
LSSAS process at system start. Once loaded into the LSA, SSP DLLs have access to encrypted and plaintext passwords that
are stored in Windows, such as any logged-on user's Domain password or smart card PINs. 
```

Comment (w0rk3r) [reply rules/windows/credential_access_lsass_loaded_susp_dll.toml:13]: Just linting

File: rules/windows/credential_access_lsass_loaded_susp_dll.toml [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+index = ["logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License v2"
+name = "Suspicious Module Loaded by LSASS"
+note = """## Setup
+If enabling an EQL rule on a non-elastic-agent index (such as beats) for versions <8.2, events will not define `event.ingested` and default fallback for EQL rules was not added until 8.2, so you will need to add a custom pipeline to populate `event.ingested` to @timestamp for this rule to work.
+"""
+references = [
+"https://blog.xpnsec.com/exploring-mimikatz-part-2/",
+"https://github.com/jas502n/mimikat_ssp"
```
Comment (w0rk3r) [inline rules/windows/credential_access_lsass_loaded_susp_dll.toml:25]:
```suggestion
    "https://blog.xpnsec.com/exploring-mimikatz-part-2/",
    "https://github.com/jas502n/mimikat_ssp"
```

**Review by w0rk3r — APPROVED** [review]
LGTM, just some lint-related comments

**Review by imays11 — APPROVED** [review]
Looks good 👍🏾

**Pull request merged**